README.md

```diff
@@ -1,5 +1,46 @@
 A [Giter8][g8] template for Akka HTTP in Scala
 
+Prerequisites:
+- JDK 8
+- sbt 0.13.13 or higher
+
+Open a console and run the following command to apply this template:
+ ```
+sbt -Dsbt.version=0.13.15 new https://github.com/akka/akka-http-java-seed.g8
+ ```
+
+This template will prompt for the following parameters. Press `Enter` if the default values suit you:
+- `name`: Becomes the name of the project.
+- `description`: Provides a description for the project.
+- `scala-version`: Specifies the Scala version for this project.
+- `akka-http-version`: Specifies which version of Akka HTTP should be used for this project.
+- `akka-version`: Specifies which version of Akka should be used for this project.
+
+This template comes with 2 different ways to implement an Akka HTTP server, along with their respective tests.
+
+- HttpServer: This is the manual way to start a server.For this case, routes are defined in separated classes: the object `com.example.routes.BaseRoutes` 
+and the trait `com.example.routes.SimpleRoutes`.
+- HttpServerHttpApp: This server is started using the new bleeding-edge `HttpApp` which bootstraps a server with just a few lines.
```

Comment:
`bleeding-edge`, what does it mean for users? not tested, I don't want that.
it will soon become obsolete, and not new any more?